knowledge-base/multiselect-disable-items.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Disable list items in the MultiSelect component.
+description: An example on how to disable list items in the KendoReact MultiSelect.
+type: how-to
+page_title: Disable items - KendoReact MultiSelect
+slug: multiselect-disable-items
+tags: disable, list, multiselect
+ticketid: 1620902
+res_type: kb
+category: knowledge-base
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>5.17.0</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® KendoReact</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description  
+
+How can I disable specific items based on a certain condition?
```

Comment:
We can add examples for "certain conditions", so the KB info will be more descriptive.